tensorflow/core/util/autotune_maps/conv_parameters.cc

```diff
@@ -94,7 +94,7 @@ MatmulParameters::MatmulParameters(
   proto_.set_c_dtype(c_dtype);
 
   proto_.set_trans_a(trans_a);
-  proto_.set_trans_b(trans_a);
+  proto_.set_trans_b(trans_b);
```

Comment:
This looks like a bug fix that can land in a separate PR, together with a regression test?

Reply:
makes sense, I can remove this change and we will add it as a separate PR